utils-internal/src/nativeCommonMain/kotlin/com/badoo/reaktive/utils/atomic/AtomicReference.kt

```diff
@@ -5,7 +5,7 @@ import com.badoo.reaktive.utils.InternalReaktiveApi
 @InternalReaktiveApi
 actual class AtomicReference<T> actual constructor(initialValue: T) {
 
-    private val delegate = kotlin.native.concurrent.AtomicReference(initialValue)
+    private val delegate = kotlin.concurrent.AtomicReference(initialValue)
```

Comment:
Do we still need our implementation of `AtomicReference` if Kotlin's one is multiplatform now?
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.concurrent/-atomic-reference/

Reply:
It seems they are still available only on Native.